cheat.php

```diff
@@ -58,10 +64,14 @@
 
 if( strlen( $Token ) !== 32 )
 {
-	Msg( 'Failed to find your token. Verify token.txt' );
+	Msg( 'Failed to find your token. Verify ' . $TokenFile );
 	exit( 1 );
 }
 
+if (isset($TokenFile)) {
```

Comment:
If it's always set, the if is unnecessary.

Reply:
And now the if is off